files/cloud-init/nfs/cloud-config

```diff
@@ -56,8 +56,9 @@ runcmd:
   #
   # Update /etc/exports - NOTE: The CIDR provided works for the whole VPC
   #
-  - echo "/export         ${aks_cidr_block}(rw,no_root_squash,async,insecure,fsid=0,crossmnt,no_subtree_check)" >> /etc/exports
-  - echo "/export         ${misc_cidr_block}(rw,no_root_squash,async,insecure,fsid=0,crossmnt,no_subtree_check)" >> /etc/exports
+  - if [ ${aks_cidr_block} == ${misc_cidr_block} ]; then echo "/export         ${aks_cidr_block}(rw,no_root_squash,async,insecure,fsid=0,crossmnt,no_subtree_check)" >> /etc/exports; else echo "/export         ${aks_cidr_block}(rw,no_root_squash,async,insecure,fsid=0,crossmnt,no_subtree_check)" >> /etc/exports && echo "/export         ${misc_cidr_block}(rw,no_root_squash,async,insecure,fsid=0,crossmnt,no_subtree_check)" >> /etc/exports; fi
+  # - echo "/export         ${aks_cidr_block}(rw,no_root_squash,async,insecure,fsid=0,crossmnt,no_subtree_check)" >> /etc/exports
+  # - echo "/export         ${misc_cidr_block}(rw,no_root_squash,async,insecure,fsid=0,crossmnt,no_subtree_check)" >> /etc/exports
```

Comment:
The code snipped from the AWS solution seems a bit cleaner as it combines the cidr_blocks and then combines those. If they are the same, then only 1 entry:

```
  #
  # Update /etc/exports
  #
  - for cidr_block in ${public_subnet_cidrs} ${private_subnet_cidrs}
  - do
  -   echo "/export         $cidr_block(rw,no_root_squash,async,insecure,fsid=0,crossmnt,no_subtree_check)" >> /etc/exports
  - done
```
The code example can be found [here](https://github.com/sassoftware/viya4-iac-aws/blob/main/files/cloud-init/nfs/cloud-config#L56-L62)